scripts/algokit-docs.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+# SETUP
+PATH_TO_SOURCE_REPOS=$1 
+# ensure the following repos are checked out to the proper release/branch for doc generation and
+# ensure each is a subdirectory of $PATH_TO_SOURCE_REPOS 
+# git clone https://github.com/algorandfoundation/algokit-cli
+# git clone https://github.com/algorandfoundation/algokit-utils-ts
+# git clone https://github.com/algorandfoundation/algokit-utils-py
+# git clone https://github.com/algorandfoundation/algokit-client-generator-py
+# git clone https://github.com/algorandfoundation/algokit-client-generator-ts
+
+# REMOVE EXISTING ALGOKIT CONTENT
+rm -rd ../docs/get-details/algokit
+
+# RENDER ALGOKIT-CLI
+cp -R $PATH_TO_SOURCE_REPOS/algokit-cli/docs/ ../docs/get-details/algokit
+mv ../docs/get-details/algokit/algokit.md ../docs/get-details/algokit/index.md
+sed -i '' "1s/.*/# Overview/" ../docs/get-details/algokit/index.md
```

Comment:
keep in mind sed won't work on windows (unless wsl) so this script will be specific to unix like stuff. 

i'd highly recommend organizing the below sed commands into functions, at least organized by the name of the repo on which changes are being performed, otherwise maintaining this long will rather hard. 

A better option might be to rewrite this into python and integrate into existing ci cd that i observed being used on this repository.